ci-operator/config/openshift/release-tests/openshift-release-tests-master.yaml

```diff
@@ -62,6 +62,13 @@ tests:
   steps:
     test:
     - ref: release-qe-tests
+- as: qe-arm-release-gate-test
+  cron: 15 * * * *
```

Comment:
I think we'd better to make the schedule different with amd64 job, e.g. every 90 mins

Reply:
I changed it every 2h.